design/openid_configuration.go

```diff
@@ -0,0 +1,95 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+// configuration represents openid-configuration
+
+var openIDConfiguration = a.MediaType("application/vnd.openIDConfiguration+json", func() {
+	a.UseTrait("jsonapi-media-type")
+	a.TypeName("OpenIDConfiguration")
+	a.Description("Indentity Provider Configuration. It list all endpoints supported by Auth Service")
+	a.Attributes(func() {
```

Comment:
Why the huge list ? Do we support these?

Reply:
https://github.com/fabric8-services/fabric8-auth/pull/277/commits/990775e3232a612d4ae66d74889b7da8e5450903